### PR TITLE
Add AdaCamp

### PR DESCRIPTION
**Pull request by harsh183**
Based on https://github.com/cristianoliveira/awesome4girls/issues/51

@cristianoliveira

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   * [Documentaries/Movies](#documentariesmovies)
   * [Jobs](#jobs)
   * [Speakers](#speakers)
+* [Orgranization toolkits](#organization-toolkits)
```
Comment (cristianoliveira) [inline README.md]: Orgranization toolkits -> Organization toolkits

File: README.md [hunk 2]
```diff
@@ -465,6 +466,13 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   - [Rewriting the Code](http://rewritingthecode.org)
     The Rewriting the Code Fellowship is a year-long program that provides the opportunity to intern in a top tech company across a variety of industries and gain essential personal and confidence-building skills needed to excel in the tech industry.
 
+### Orgranization toolkits
```
Comment (cristianoliveira) [inline README.md]: Orgranization toolkits -> Organization toolkits

**Comment by cristianoliveira** [comment]
Hey @harsh183 

Looks like the summary is not in the right order. Could you address it?

"Events/Meetups", "General", "Initiatives/Programs", "Orgranization toolkits", "Other Initiatives", "Other Lists"

**Comment by harsh183** [comment]
@cristianoliveira What do you mean? What is the issue with order?

**Pull request merged**